openedx.yaml

```diff
@@ -1,13 +1,17 @@
 # This file describes this Open edX repo, as described in OEP-2:
 # http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
 
-owner: tuchfarber
-supporting_teams: edx/enterprise
+owner: schenedx
+supporting_teams:
+  - masters-devs
+  - edx/enterprise
 nick: desg
+tags:
+  - webservice
 oeps:
   oep-2: True
   oep-7: True
+  oep-10: True
   oep-18: True
   oep-19: True
 openedx-release: {ref: master}
```

Comment:
@adamstankiewicz @brittneyexline Should this repo be tagged as part of Open edX releases? I would have thought no.

Reply:
are you asking if this should be made available as part of the next official openedx release? given the current use case as well as the ambiguity of ownership or future development here, i would tend to say this isn't something that would be useful or relevant to the open source community at this point.

Reply:
I would agree.  So this line needs to be removed.

Reply:
Sure, I can remove it. thanks for the context.

Reply:
Updated.